### PR TITLE
feat: Devgenesis: Validator replacing script

### PR DESCRIPTION
**Pull request by MissingNO57**

### CODE REVIEW

File: scripts/genesis_helpers.py [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+
+def get_validator_info(genesis, validator_pubkey):
+    for val in genesis["validators"]:
+        if val["pub_key"]["value"] == validator_pubkey:
+            return val
+
+    assert True, "Validator not found in genesis"
+
+
+def replace_validator_from_pubkey(
```
Comment (MissingNO57) [inline scripts/genesis_helpers.py:130]: This function we care about

File: scripts/Pipfile [hunk 2]
```diff
@@ -5,6 +5,7 @@ name = "pypi"
 
 [packages]
 bech32 = "==1.2.0"
+cosmpy = "*"
```
Comment (pbukva) [inline scripts/Pipfile:8]:
```suggestion
cosmpy = "==0.9.2"
```

**Review by pbukva — APPROVED** [review]
LGTM

File: scripts/replace_validator.py [hunk 3]
```diff
@@ -106,41 +194,18 @@ def main():
...
-    )
-
-    # Convert operator base account address
-    genesis_dump = re.sub(
-        target_operator_base_address,
-        dest_operator_base_address,
-        genesis_dump,
-    )
-
-    genesis = json.loads(genesis_dump)
```
Comment (pbukva) [inline scripts/replace_validator.py]: This was replaced due to potential risk replacing something what was not supposed to be replaced + performance impact.

**Review by pbukva — APPROVED** [review]
LGTM

**Review by pbukva — APPROVED** [review]
LGTM

**Pull request merged**